src/inference/src/ie_core.cpp

```diff
@@ -860,6 +870,10 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
                         "You can only get_config of the AUTO itself (without devices). "
                         "get_config is also possible for the individual devices before creating the AUTO on top.");
 
+        if (name == ov::force_tbb_terminate.name()) {
+            const auto flag = executorManager()->getTbbFlag();
+            return decltype(ov::force_tbb_terminate)::value_type(flag);
+        }
```

Comment:
I suppose for such needs we need to introduce `get_property` without device name, since such property is Core specific, not device-specific.
Can we do this?

Reply:
Yes, It has been resolved in https://github.com/openvinotoolkit/openvino/pull/12022

Reply:
In that PR you use `set_property` with no device name, but I think we can also add `get_property` with no device name to have mirrored API.